lib/assets/javascripts/cartodb/old_common/header.js

```diff
@@ -475,6 +477,16 @@ cdb.admin.Header = cdb.core.View.extend({
   _onTabClick: function(e) {
     e.preventDefault();
     window.table_router.navigate(this._generateTableUrl(e), {trigger: true});
+  },
+
+
+  _doExport: function(e) {
+    this.mapTab._doExport(e);
```

Comment:
I'm a little hesitant to mark the functions as public because that could give an air of respectability to what's going on here, I actually want to lampshade that this is a hack that's only here because the right solution requires a disproportionate amount of effort (We would have to go through the MapTab view logic and rip out a lot of logic that could result in who knows how many problems, plus it would make merging in future upstream changes a challenge). Marking them as public because they're being access publicly may be the right thing to do, but in that case I'd want to make it abundantly clear that this is bad and I should feel bad.